Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
-gem 'cocoapods', '1.6.0.beta.1'
+gem 'cocoapods', '1.7.0.rc.2'
```

Comment:
#pp Keep this on a GA version, I think the latest is 1.6.1 (unless we need something forthcoming in 1.7.0?)

Reply:
Switched to 1.7 because they’ve resolved the backwards compatibility issues by including a `swift_version` in the published podspec. The `swift_versions` flag in this version was the best way to express support for 4.2 and 5.0. Plus, it’s in keeping with tradition to publish BonMot with prerelease versions of CocoaPods! :trollface:

Reply:
I see... My #pp there is to split something like that into a WIP branch that tracks the dependency until its GA :) Up to you.